ts-rs/src/lib.rs

```diff
@@ -397,7 +397,10 @@ impl Dependency {
         let exported_to = T::get_export_to()?;
         Some(Dependency {
             type_id: TypeId::of::<T>(),
-            ts_name: T::name(),
+            ts_name: match T::name() {
+                x if !x.contains('<') => x,
+                x => x.split('<').next().unwrap().to_owned()
+            },
```

Comment:
Dependency seems to need the name without any generics, so the same hacky approach was used here